software/ompi/v4.0/timeline-graph.php

```diff
@@ -34,6 +34,7 @@
 milestone("v4.0.3", "2020-03-04", $data, $vpos);
 milestone("v4.0.4rc1", "2020-05-08", $data, $vpos);
 milestone("v4.0.4rc2", "2020-05-22", $data, $vpos);
+milestone("v4.0.4rc2", "2020-06-09", $data, $vpos);
```

Comment:
do you mean to have v4.0.4rc3 here?

Reply:
oops.  yes.  Thanks!  I've fixed and repushed.